brooklyn-docs/guide/start/blueprints.md

```diff
@@ -40,26 +43,54 @@ location:
     credential: s3cr3tsq1rr3ls3cr3tsq1rr3ls3cr3tsq1rr3l
 {% endhighlight %}
 
-**Note**: See __[Locations](../ops/locations)__ in the Operations section of the User Guide for instructions on setting up alternate cloud providers, bring-your-own-nodes, or localhost targets, and storing credentials/locations in a file on disk rather than in the blueprint.
+Or, if you already have machines provisioned, you can use the "bring your own nodes" (byon) approach. 
+Of course, replace the identity and address values below with your own values.
+{% highlight yaml %}
+location:
+  byon:
+    user: myuser
+    password: mypassword
+    # or...
+    #privateKeyFile: ~/.ssh/my.pem
+    hosts:
+    - 192.168.0.18
+    - 192.168.0.19
+{% endhighlight %}
+
+**Note**: See __[Locations](../ops/locations)__ in the Operations section of the User Guide for instructions on setting
+up alternate cloud providers, bring-your-own-nodes, or localhost targets, and storing credentials/locations in a file 
+on disk rather than in the blueprint.
+(For the application above, if you are using a "Bring your own Nodes" location, you will need at least three nodes.)
+
+First you will have to log in to brooklyn:
+{% highlight bash %}
+$ br login http://localhost:8081/
+{% endhighlight %}
 
-With the modified YAML in the dialog, click "Finish". The dialog will close and Brooklyn will begin deploying your
-application. Your application will be shown as "Starting" on the web console's front page.
+To secure the server you can add a username and password in Brooklyn's properties file, as described in the User Guide. 
+Then the login command will require the additional parameters of the userid and password.
 
-Depending on your choice of location it may take some time for the application nodes to start, the next page describes how you can monitor the progress of the application deployment and verify its successful deployment.
+Now you can create the application with the command below:
 
-### Launching from the Catalog
+{% highlight bash %}
+$ br deploy myapp.yaml
+Id:       hTPAF19s   
+Name:     Tomcat   
+Status:   In progress   
+{% endhighlight %}
 
-Instead of pasting the YAML blueprint each time, it can be added to Brooklyns Catalog where it will be accessible from the Catalog tab of the Create Application dialog.
+Depending on your choice of location it may take some time for the application to start, the next page describes how 
+you can monitor the progress of the application deployment and verify its successful deployment.
 
-[![Viewing Catalog entries in Add Application dialog.](images/add-application-catalog-web-cluster-with-db.png)](images/add-application-catalog-web-cluster-with-db-large.png)
 
-<!-- TODO: more detail for adding to catalog? but wait for persistence to be the default, 
-     rather than extensively document default.catalog.bom.
-     also need to include instructions on stopping (currently in help, including stopping apps) -->
+## Launching from the Catalog
```

Comment:
I am not sure if this sub-section is really fits here anymore:
- Using the deployy command, we are not doing a copy/paste operation each time a blueprint is deployed.
- We don't handle deploying from a catalog entry using the CLI
Options seem to be:
- Remove this sub-section altogether; OR
- For complex blueprints, it is preferable to store them as catalog items, so that they are available for multiple users and can be referenced by more complex blueprints.